src/operator/tensor/elemwise_binary_broadcast_op_extended.cc

```diff
@@ -33,7 +44,33 @@ NNVM_REGISTER_OP(_backward_broadcast_power)
                                                               mshadow_op::power_rgrad>);
 
 MXNET_OPERATOR_REGISTER_BINARY_BROADCAST(broadcast_maximum)
-.describe(R"code(Element-wise maximum of array elements with broadcasting.
+.describe(R"code(Returns element-wise maximum of the input arrays with broadcasting.
+
+It compares input arrays and returns the new array having the element-wise maxima.
+
+Example::
+
+   x = [[ 1.,  1.,  1.],
+        [ 1.,  1.,  1.]]
+
+   y = [[ 0.],
+        [ 1.]]
+
+   broadcast_maximum(x, y) = [[ 1.,  1.,  1.],
+                              [ 1.,  1.,  1.]]
+
+If any array contains elements as `inf` then, array with `inf` is returned with broadcasting as output.
```

Comment:
comma comes before then, not after then. "If ____, then ____". This mistake happens in many places, try to be attentive to this.